bedrock/cms/middleware.py

```diff
@@ -33,6 +33,9 @@ def __call__(self, request):
         response = self.get_response(request)
 
         if response.status_code == HTTPStatus.NOT_FOUND:
+            if self._run_null_byte_check(request) is True:
```

Comment:
I think this might read a bit nicer if it was something like `if self._has_null_byte`. When I first read it I was wondering what `True` means... that it found a null byte or that it didn't find one.

Reply:
Sounds good, yeah